assets/styles/generic/_media.scss

```diff
@@ -6,53 +6,52 @@
 // 1. Setting `vertical-align` removes the whitespace that appears under `img`
 //    elements when they are dropped into a page as-is. Safer alternative to
 //    using `display: block;`.
-//
+
 audio,
 canvas,
 iframe,
 img,
 svg,
 video {
-    vertical-align: middle; /* [1] */
+    vertical-align: middle; // [1]
 }
 
 //
 // Add the correct display in iOS 4-7.
-//
+
```

Comment:
I suggest removing the leading `//` if we are removing the trailing `//` everywhere.

```diff
- //
  // Add the correct display in iOS 4-7.
```